easybuild/easyconfigs/o/OrfM/OrfM-0.6.1-foss-2016b.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     LGPL-v3.0
+#
+# Notes::
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'OrfM'
+version = '0.6.1'
+namelower = 'orfm'
```

Comment:
@robqiao You don't need this, the `%(namelower)s` template is resolved by EasyBuild based on `name` automatically